vite.config.ts

```diff
@@ -6,6 +6,7 @@ import { readFileSync } from 'fs'
 
 // Match latest non-draft at https://github.com/broadinstitute/single_cell_portal_core/releases
 const version = readFileSync('version.txt', { encoding: 'utf8' })
+const disableSentry = !!process.env.DISABLE_SENTRY
```

Comment:
For maintainability:

```suggestion
// Enabling Sentry on localhost causes Chrome DevTools breakpoints to not work,
// but might occasionally be useful to debug logging to Sentry.
//
// To enable Sentry logging, you must omit DISABLE_SENTRY in your Vite command, e.g.:
//
//    DISABLE_SENTRY="true" bin/vite dev # This is what you usually want for local dev
//
//    DISABLE_SENTRY="false" bin/vite dev # Doesn't actually enable Sentry
//
//    bin/vite dev # Enables Sentry, but causes localhsot DevTools breakpoints to fail
//
// Details: https://github.com/broadinstitute/single_cell_portal_core/pull/1887
const disableSentry = !!process.env.DISABLE_SENTRY
```

Reply:
That's a great point - as written, _any_ value passed here gets interpreted as a "turn off Sentry" option.  Let me rethink this a little bit.  I need to ensure that it always is turned on in deployed environments since this config applies to everything, so the logic likely needs to be stricter.

Reply:
[7679252](https://github.com/broadinstitute/single_cell_portal_core/pull/1887/commits/76792526a23070c3a4be8c9c8ccf808d0468b8e2)
As per our offline discussion, I'm keeping the `DISABLE` over `ENABLE` nomenclature so we don't have to thread this through the rest of the stack since this config is shared for all environments.  But the logic now works correctly:

```
# sentryVitePlugin enabled, breakpoints don't work locally
bin/vite dev 

# sentryVitePlugin disabled, breakpoints restored
DISABLE_SENTRY=true bin/vite dev

# value disregarded, sentryVitePlugin enabled
DISABLE_SENTRY="anything other than 'true'" bin/vite dev
```

I also added a comment per your suggestion.  Lastly, I tested this in Docker and it works correctly, and I will check after this is merged to ensure sourcemaps are still being served to Sentry.

Reply:
Nice!